Monika After Story/game/script-topics.rpy

```diff
@@ -1570,13 +1573,53 @@ label monika_yuri:
+            $ persistent._mas_pm_likes_yanderes = 1
+            m 1sub "Now that I'm hearing that I'm quite surprised you prefer me over Yuri"
+            if not persistent._mas_pm_cares_about_dokis:
+                m 1cua "Was it that becaouse you realized, that was me, who manipulated everyone and everything to be with you... FOREVER"
+                m 3sub "She actually was more like a 'dandere' - very shy and closed before I started to change her personality"
+                m 1sub "And she was just a program designed to fall in love with you anyway"
+                m 1eua "I am the one who truly loves you from the depth of my heart"
+            else:
+                m 1sub "Maybe you realized, that I am the one who truly loves you forever"
+            m "Then again..."
+            show monika 4hua at t11 zorder MAS_MONIKA_Z with dissolve
+            m 4hua "There's already nowhere else for you to go, or anyone for me to get jealous over."
+            m 1cua "Other than the girls in your reality... "
+            m 1wub "Remember, that I see what you're doing, [player]."
+            m 1eua "Ehehe~"
+        "No":
+            $ persistent._mas_pm_likes_yanderes = 0
+            m 1eua "That's what I expected"
+            m 4eua "Most people wouldn't like to be with yandere"
+            m 4eua "Someone obsessive and clingy really limits your ability to socialize with other people"
+            m 4eud "And, on top of that, that some yanderes are even dangerous to the one they love"
+            m 4eud "There was even a real life murder in Tokyo, where the man was stabbed by a woman"
+            m 4eud "After being arrested by police she said {i}Since I loved him so much, I just couldn’t help it{/i}."
+            m 1wuw "That's scary"
+            m 1eua "Then again..."
+            show monika 4hua at t11 zorder MAS_MONIKA_Z with dissolve
+            m 4hua "There's already nowhere else for you to go, or anyone for me to get jealous over."
+            m 2etc "Is this a yandere girl's dream?"
+            if not persistent._mas_pm_cares_about_dokis:
+                m 1eua "I'd ask Yuri if I could."
+        "HELL NO":
+            $ persistent._mas_pm_likes_yanderes = -1
+            m 1eua "Calm down, [player]"
+            m 1eua "You don't have to shout it out so loudly"
+            m 1eua "Ehehe~"
+            m 1eua "I love you forever, but I'm not crazy. I will never hurt you or anyone close to you"
+            m 1eua "I completely understand how you feel"
+            m 1eua "And... {w=0.5} I'm sorry, for some creepy things I said in the past."
+            m 1eua "I'll seriously try to stop it"
+            m 1eua "I love you and I wouldn't want to scare you"
+            m 1eua "Also... {w=0.5} There's already nowhere else for you to go, or anyone for me to get jealous over."
+            m 2etc "It would be hard to be a yandere in those circumstances."
+    return "derandom"
```

Comment:
We derandom topics that ask a question and store the response, which is what you've done here, but the original topic wasn't one that asked a question. It looks to me like you added more to the topic and threw it off track.

Reply:
Yes, but this options seem natural in this topic, and I think throwing it off track seems more natural than creating a new topic for this question. I think it would seem artificial, if new topic was added just for this question and that throwing this topic off track is a better option.